cvat-ui/src/components/create-task-page/advanced-configuration-form.tsx

```diff
@@ -16,6 +17,14 @@ import patterns from 'utils/validation-patterns';
 
 const { Option } = Select;
 
+export enum SortingMethods {
```

Comment:
What is a reason to clone the definition in several places? Also I will prefer 'default', 'native', 'random', ... It looks better.

Reply:
One in `cvat-core` and one in `cvat-ui`